tests/fileTypeTests.py

```diff
@@ -123,6 +124,13 @@ def test_award_fin_mixed(self):
         jobId = self.jobIdDict["awardFinMixed"]
         self.passed = self.run_test(
             jobId, 200, "finished", 17206, 5, "complete", 87, True)
+        # Test that whitespace is converted to null
+        rowThree = self.interfaces.validationDb.session.query(AwardFinancial).filter(AwardFinancial.row_number == 3).filter(AwardFinancial.submission_id == self.interfaces.jobDb.getSubmissionId(jobId)).first()
```

Comment:
@jworcestBAH Just to clarify, I don't think this point needs to hold up merging this. Was more fishing around to see if there's an easy way to replicate the unittest functionality without the row number. If there's not, there's not. Also, I anticipate that we'll have to take a pass through our unit tests in the near future anyway, as we move to real data.